### ISSUE TITLE
Calendar does not build on 2.3

### ISSUE THREAD

**Issue by Mackiovello** [StarcounterApps/Calendar#31]
The app fails on the following error:
```
C:\Users\Erlend\Documents\apps\Calendar\Calendar.sln.metaproj : error MSB3202: The project file "C:\Users\Erlend\Docume
nts\apps\Calendar\..\PropertyMetadata\Smorgasbord.PropertyMetadata.csproj" was not found. [C:\Users\Erlend\Documents\ap
ps\Calendar\Calendar.sln]
C:\Users\Erlend\Documents\apps\Calendar\Calendar.sln.metaproj : error MSB3202: The project file "C:\Users\Erlend\Docume
nts\apps\Calendar\..\..\StarcounterSamples\Simplified\Simplified.csproj" was not found. [C:\Users\Erlend\Documents\apps
\Calendar\Calendar.sln]
```

When this is fixed, `develop` should be merged into `master`.

**Comment by andwah** [comment]
You also need to pull the `StarcounterSamples\Simplified` project from github. (maybe the path has been changed?)

and im not sure why `Smorgasbord.PropertyMetadata` is still referenced as a project.. `Smorgasbord.PropertyMetadata` is a nuget and should be used instead.

**Comment by andwah** [comment]
Here is a link to PropertyMetadata  https://www.nuget.org/packages/Smorgasbord.PropertyMetadata/

**Comment by Mackiovello** [comment]
> You also need to pull the StarcounterSamples\Simplified project from github. (maybe the path has been changed?)

My understanding is that Simplified is included in Starcounter.

I'll reference the NuGet package.

**Comment by andwah** [comment]
> My understanding is that Simplified is included in Starcounter.

Your right. i dont know where my head was :-)

**Comment by Mackiovello** [comment]
Fixed with 	77012f7
